### DISCUSSION TITLE
Outdated pdfjs-dist version being used

### DISCUSSION THREAD

**Discussion by chefinDan** [wojtekmaj/react-pdf#1488] [Q&A] [answered]
The latest version of this package uses pdfjs-dist v2.16.105. this causes problems when needing types for PDFDocumentProxy, as react-pdf doesn't export the types it uses from pdfjs-dist. 
The package.json in this repo shows the latest version of pdfjs-dist, but the same version from npm uses the old version..am I missing something?

**Comment by wojtekmaj** [comment, marked as answer]
What you're missing is that the most recent version of React-PDF was NOT native TypeScript and external, incorrect TS types are likely used. On main, we currently have unreleased version 7 which IS native TypeScript, so this problem should not occur once it's released.
